samples/samples_for_tutorials/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "grs-samples",
+  "description": "NodeJS code samples for Google Retails Search API",
+  "license": "Apache-2.0",
+  "author": "Google LLC",
+  "engines": {
+    "node": ">=10"
+  },
+  "files": [
+    "*.js"
+  ],
+  "private": true,
+  "scripts": {
+    "test": "mocha --timeout 600000 test/*.test.js"
+  },
+  "dependencies": {
+    "@google-cloud/bigquery": "^5.9.2",
```

Comment:
if you create the samples in a subdirectory like this, they won't be automatically added to README.md and samples/README.md, and it makes the library deviate from how others are layed out.

I would instead add the samples in `samples/`, then you can take advantage of the logic that generates the README, also the top level `package.json`.

Reply:
Is it possible that some other code samples, not used by the interactive tutorials, would be added to the same samples/ directory?

If it is possible, I would prefer to store our samples separately. The Tutorial's user will open the repository in the CloudShell and run them using their own project and prepared data set. If some other code samples, for example those which are prepared as static code samples built in the documentation, will be stored along with interactive cloud shell samples, it will confuse the users.